packages/opensrp-plans/src/components/NewPlan/index.tsx

```diff
@@ -0,0 +1,73 @@
+/** component renders view where users can create plans
+ */
+import React from 'react';
+import {
+  defaultInitialValues,
+  PlanForm,
+  PlanFormFieldsKeys,
+  processActivitiesDates,
+} from '@opensrp/plan-form';
+import { Layout, PageHeader } from 'antd';
+import Helmet from 'react-helmet';
+import { CommonProps, defaultCommonProps } from '../../helpers/common';
+import { CREATE_PLAN } from '../../lang';
+import { PLANS_LIST_VIEW_URL } from '../../constants';
+import {
+  defaultEnvConfig,
+  getFormActivities,
+  getPlanActivitiesMap,
+  InterventionType,
+  planActivities,
+} from '@opensrp/plan-form-core';
+
+type CreatePlanViewProps = CommonProps;
+
+const defaultProps = {
+  ...defaultCommonProps,
+};
+
+/**
+ * CreateProductView component
+ *
+ *  props - component props
+ */
+
+const CreatePlanView = (props: CreatePlanViewProps) => {
+  const { envConfigs, baseURL } = props;
+  const pageTitle = CREATE_PLAN;
```

Comment:
For EUSM is this the correct string to use here?

Reply:
This was going to be subject to translations so that we can have the correct english translations of the project, I can change have it to be EUSM-specific, untill translations is fully scoped.

Reply:
Please defer to @cafootitt and @dubdabasoduba on whether you should do that

Reply:
@p-netm  I agree with you. I guess it just increases the amount of work to write some of these constants once the MLS work is ready.